src/docs/build-posix.dox

```diff
@@ -21,7 +21,7 @@ We also suggest <a href="https://ninja-build.org/">Ninja</a> and
 First, clone the repository:
 
 @code
-git clone git://github.com/wiredtiger/wiredtiger.git
+git clone https://github.com/wiredtiger/wiredtiger.git
```

Comment:
Take this as a note and not an addressable comment. I believe `https://` cloning was deprecated a while back, so developers that use `https://` to clone on their workstations don't have permission to push branches to GitHub.

I don't think it's an issue for external contributors but I'm curious if this will impact new WT devs.
I have a new MacBook I'll set up in a week or so. I can investigate this then. Feel free to merge in the meantime

Reply:
Good point. I assume this public doc is mainly targeted at external contributors, who may run into issues with SSH clone (as they are not members of the repo). I double-checked our internal wiki and it does write SSH clone (which is good).